tests/lessons/lesson02.js

```diff
@@ -1,8 +1,4 @@
 window.onload = function() {
```

Comment:
I think these files for playing a specific lesson are useless now that we have the course overview. If you agree, you can delete both this and the corresponding html (also from lesson 1 if they're still there). The old /src/lessons/lesson02.js that implemented the game before should also be delete, since it won't work anymore anyway (same for /src/lessons/lesson01.js if it's still there).